### ISSUE TITLE
[Anon] Bug report: Death - Ossiarch Bonereapers.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/warhammer-age-of-sigmar#1734]
**File:** Death - Ossiarch Bonereapers.catz

**BattleScribe version:** 2.03.25

**Platform:** Android

**Dropbox:** No

**Description:** When we take two characters named as leader, example: nagash and arkhan or katakros, we cannot put the mandatory legion artefact and there is therefore no need to put it and so there should be no error.

**Comment by cartag** [comment]
The artefact doesn't have to go on your general and can be taken by another leader in your army.

**Comment by oggurt-da-bog-zombie** [comment]
I think I'm experiencing a similar or even the same issue as the OP, [I mentioned this in a reddit post here](https://www.reddit.com/r/ageofsigmar/comments/m7dz71/questions_on_1000pt_lists/).

TL;DR when taking Vokmortian as the only leader in a 1000pt list, Battlescribe shows an error for list validation as there isn't anyone to give the Mortis Praetorians artefact to (or whatever the "Artificers Blade" is).

**Comment by cartag** [comment]
You can just ignore that warning, it would pop up for any army that has only a singular named Leader unit in the roster. I'm not sure if there's a real way to get around that error popping up, it's a restriction of the Battlescribe platform itself.
